tools/tune/eq/eq_iir_blob_pack.m

```diff
@@ -39,7 +44,7 @@
 nbytes_data = nbytes_head + nbytes_coef;
 
 %% Get ABI information
-[abi_bytes, nbytes_abi] = eq_get_abi(nbytes_data);
```

Comment:
At line 22, if `bs` output is not needed every time, please consider removing or suppressing `bs`

Reply:
Yep, no need for it.